test/test_functional_tensor.py

```diff
@@ -19,6 +22,21 @@ def test_hflip(self):
         self.assertEqual(hflipped_img.shape, img_tensor.shape)
         self.assertTrue(torch.equal(img_tensor, hflipped_img_again))
 
+    def test_crop(self):
+        img_tensor = torch.FloatTensor(3, 16, 16).uniform_(0, 1)
+        top = random.randint(0, 15)
+        left = random.randint(0, 15)
+        height = random.randint(1, 16 - top)
+        width = random.randint(1, 16 - left)
+        img_cropped = F_t.crop(img_tensor, top, left, height, width)
+        img_PIL = transforms.ToPILImage()(img_tensor)
+        img_PIL_cropped = F.crop(img_PIL, top, left, height, width)
+        img_cropped_GT = transforms.ToTensor()(img_PIL_cropped)
+
+        max_diff = (img_cropped_GT - img_cropped).abs().max()
+
+        self.assertLess(max_diff, 5e-3, "functional_tensor crop not working")
```

Comment:
Can't we make the difference 0? Is this because we need to perform casting between types, which gives some accuracy differences?
In that case, can't you directly make `img_tensor` be a `uint8` tensor with values from 0-255?

Reply:
Done!